py/picca/cf.py

```diff
@@ -279,6 +280,95 @@ def compute_xi_forest_pairs(z1, r_comov1, dist_m1, weights1, delta1, z2,
     return (rebin_weight, rebin_xi, rebin_r_par, rebin_r_trans, rebin_z,
             rebin_num_pairs)
 
+@jit(nopython=True)
+def compute_xi_forest_pairs_fast(z1, r_comov1, dist_m1, weights1, delta1, z2,
```

Comment:
why don't we replace this function with compute_xi_forest_pairs? I mean, is there a particular reason as to why we would like to keep the slow version? Similarly for the other updated functions